frontends/web/src/routes/account/summary/accountssummary.tsx

```diff
@@ -81,19 +81,27 @@ export const AccountsSummary = ({
     if (summary.success) {
       setSummaryData(summary.data);
     } else {
-      console.error(summary.error);
+      if (summary.errorMessage === 'syncInProgress') {
```

Comment:
this is not an error message but an error code, so I'd expect `summary.errorCode === 'syncInProgress'`